### PR TITLE
Update namespaces for RuboCop 0.78

### PR DESCRIPTION
**Pull request by helariL**
Last 10 minor versions of RuboCop have introduced multiple breaking
changes by renaming the in-use namespaces and also added couple of
deprecation warnings. This commit addresses all of those and bumps
the gem version to 0.0.4

Related `rubocop` changelog entry - https://github.com/rubocop-hq/rubocop/blob/master/CHANGELOG.md#changes-11

### CODE REVIEW

**Comment by sviik** [comment]
`Updated` -> `Update`

**Comment by helariL** [comment]
> `Updated` -> `Update`

Copy pasted from the previous **PR**. Changed nonetheless @sviik

File: rubocop-salemove.gemspec [hunk 1]
```diff
@@ -1,6 +1,6 @@
 Gem::Specification.new do |s|
   s.name        = 'rubocop-salemove'
-  s.version     = '0.0.3'
```
Comment (sviik) [inline rubocop-salemove.gemspec:3]: Split version bump to a separate commit.

**Comment by sviik** [comment]
Commit message would be nice, e.g. why is the change needed. I don't know much about RuboCop namespaces personally.

File: rubocop-salemove.gemspec (hunk 1 above)
Comment (helariL) [reply rubocop-salemove.gemspec:3]: b635e4

**Review by sviik — APPROVED** [review]

**Comment by helariL** [comment]
!merge
